src/containers/projects/view/user-view.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+  import UserActivityCard from '@/components/projects/view/user-activity-card.svelte';
+
+  export let projectStage;
+  export let activities;
+  export let competences;
+  export let account;
+
+  $: activityCards = activities.filter(x => !x.internal);
```

Comment:
Why does it need to be reactive?

Reply:
Not really sure, wish I could give a more precise answer to this. Maybe it's to make the interface react to changes in the activities from the outside.
Short answer: it works like this, and I wouldn't really want to go seeking different solutions right now) you can mark this spot though, perhaps we could optimize it later

Reply:
I wouldn't really call it "different solutions", it is just removing 2 symbols that add a lot of overhead in observers. wouldn't take a minute to make it works as it should.
I don't think activities are supposed to "change from the outside", especially in the user view; at least not without reloading the whole page.
If you still fear the lack of reactivity, you can just put this expression directly in the template in place of the `activityCards` variable; it is short enough.

Reply:
That's the problem, there's a lot more happening with the internal variables than you might imagine, that's why I wouldn't be so quick to make conclusions. I understand the overhead, it's just that, as of now, fixing it is dangerous. Note that most of the requests to the API on this page update the project object on their own, without rerequesting it from the backend